main.py

```diff
@@ -47,6 +42,70 @@ async def get_app(app_id:str):
         return {"message": "No App found for id"}
     except Exception as e:
         return {e}
+    
+
+@app.post("/app")
+async def create_app(app: App):
+    try:
+        document = jsonable_encoder(app)
+        document["authors"] = [ObjectId(author) for author in document["authors"]]
+        result = await db['apps'].insert_one(document)
```

Comment:
I think it can be replaced with

 document_app = await insert_one(
            collection_name="apps",
            new_data=app
        )